configs/fire_tv_events_schemas.json

```diff
@@ -0,0 +1,1545 @@
+{
+    "source": "telemetry",
+    "filters": {
+        "appName": [
+            "FirefoxForFireTV", 
+            "OTHER"
```

Comment:
Is "OTHER" needed?

Reply:
Unfortunately it is, since we never moved `FirefoxForFireTV` out of the `OTHER` bucket

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1553585. Once we backfill and that is completed we can remove `OTHER`.